src/igz_controls/directives/ng-scrollbars-config.directive.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Directive adds max-height for scrollbar container
+ * @param igzNgScrollbarsConfig.maxElementsCount (number) maximum number of children elements that fit into the scrollbar container
+ * @param igzNgScrollbarsConfig.childrenSelector (string) selector for children elements
+ */
+(function () {
+    'use strict';
+
+    angular.module('iguazio.dashboard-controls')
+        .directive('igzNgScrollbarsConfig', igzNgScrollbarsConfig);
+
+    function igzNgScrollbarsConfig($timeout, lodash) {
+        return {
+            restrict: 'A',
+            link: link
+        };
+
+        function link(scope, element, attrs) {
+            var config = angular.fromJson(attrs.igzNgScrollbarsConfig);
+
+            $timeout(function () {
+                activate();
+            });
+            scope.$watch(function () {
+                return element.find(config.childrenSelector).length
+            }, activate);
+
+            /**
+             * Constructor method
+             */
+            function activate() {
+                var childrenElements = element.find(config.childrenSelector);
+                var maxElementsCount = config.maxElementsCount;
+                var scrollbarContainerHeight = 0;
+
+                if (childrenElements.length > maxElementsCount) {
+                    scrollbarContainerHeight = lodash.chain(childrenElements)
+                                                     .take(maxElementsCount)
+                                                     .reduce(function (result, child) {
+                                                         var styles = window.getComputedStyle(child);
+                                                         var margin = parseFloat(styles['marginTop']) + parseFloat(styles['marginBottom']);
+
+                                                         return result + child.offsetHeight + margin
+                                                     }, 0)
```

Comment:
`<NON-CRITICAL>`
As much as I like `reduce`, you've implemented the `sum()` method here.
```js
function sum(numberArray) {
    return numberArray.reduce(function (prev, curr) { return prev + curr; }, 0);
}
```
For readability, I suggest this:
```js
scrollbarContainerHeight = lodash.chain(childrenElements)
    .take(maxElementsCount)
    .map(function (child) {
        var styles = window.getComputedStyle(child);
        var margin = parseFloat(styles['marginTop']) + parseFloat(styles['marginBottom']);

        return child.offsetHeight + margin;
    })
    .sum()
    .value();
```
You first map each child to a number, then you sum all numbers.
`</NON-CRITICAL>`